pipeline.md

```diff
@@ -0,0 +1,43 @@
+# Pipelines!
```

Comment:
This should probably go in the docs in the `docs/` directory. This page specifically:

https://bedrock.readthedocs.io/en/latest/pipeline.html